test/releasers/node.ts

```diff
@@ -85,18 +146,25 @@ describe('Node', () => {
         async () => {
           const expectedVersion = element.version;
           const pkgName = 'node-test-repo';
+          const github = new GitHub({
+            owner: 'googleapis',
+            repo: 'node-test-repo',
+          });
+          const mock = mockGithub({
+            github,
+            fixtures: ['package.json'],
+            notFound: [],
+          });
           const releasePR = new Node({
-            github: new GitHub({owner: 'googleapis', repo: 'node-test-repo'}),
+            github,
             bumpMinorPreMajor: element.bumpMinorPreMajor,
           });
-          mockRequest(releasePR);
-          // for Node.getPackageName
-          stubFilesToUpdate(releasePR.gh, ['package.json']);
 
           // no latestTag to pass to getOpenPROptions (never found a release)
           // releaser should set defaultInitialVersion
           const openPROptions = await releasePR.getOpenPROptions(COMMITS);
 
+          mock.verify();
```

Comment:
the important thing to check in this test file is that
1/ no assertions changed (other than adding in the `mock.verify()` calls
2/ no changes in __snapshots__/node.js

Reply:
so this serves the same purpose as `.done()` when using nock 👍 

thanks for these test improvements.